.env.example

```diff
@@ -0,0 +1,12 @@
+# app
+PORT=3000
+
+# database
+DB_HOST=localhost
+DB_PORT=3306
+DB_USER=root
+DB_PASSWORD=root
+DB_DATABASE=
+
+# jsonwebtokens
+JWT_SECRET=f9bf78b9a18ce6d46a0cd2b0b86df9da
```

Comment:
api key and jwt_secret should never be made publicly visibile.

Reply:
in real world application, we generate a command that helps to set this kinda value behind the scene